LDAR_Sim/src/virtual_world/infrastructure.py

```diff
@@ -118,18 +126,18 @@ def update_propagating_params(
                     if site_type_val is not None:
                         prop_params[pdc.Common_Params.METH_SPECIFIC][param][method] = site_type_val
 
-            # Updating propagating parameters with site info
-            for param in IC.Sites_File_Constants.PROPAGATING_PARAMS:
-                site_val = site_row_df_info.get(param, None)
-                if site_val is not None:
-                    prop_params[param] = site_val
+        # Updating propagating parameters with site info
```

Comment:
Why did the indentation change here?

Reply:
This was a bug fix. 
The sites file specific propagating parameters should always happen. They shouldn't be limited by if the site type file exists.